stone/cli.py

```diff
@@ -368,6 +367,17 @@ def main():
         # easier to do debugging.
         return api
 
+def _load_module(name, path):
+    file_name = os.path.basename(path)
+    module_name = file_name.replace('.', '_')
+
+    module_specs = importlib.util.spec_from_file_location(module_name, path)
+    module = importlib.util.module_from_spec(module_specs)
+    module_specs.loader.exec_module(module)
+
+    sys.modules[name] = module
+
+    return module
 
```

Comment:
This doesn't work for python <= 3.5, if you want to preserve the ability to run those lower versions, you can borrow inspiration from https://github.com/epfl-scitas/spack/blob/a60ae07083a5744607064221a0cd48204a54394e/lib/spack/llnl/util/lang.py#L598-L625